src/sonic-config-engine/tests/sample_output/frr.conf

```diff
@@ -29,93 +31,55 @@ link-detect
 interface PortChannel04
 link-detect
 !
-!
-! Set ip source to loopback for bgp learned routes
-route-map RM_SET_SRC permit 10
-    set src 10.1.0.32
-!
- 
-route-map RM_SET_SRC6 permit 10
-    set src fc00:1::32
-!
-ip protocol bgp route-map RM_SET_SRC
-!
-ipv6 protocol bgp route-map RM_SET_SRC6
-!
 !!
 !
 ! set static default route to mgmt gateway as a backup to learned default
 ip route 0.0.0.0/0 10.0.0.1 200
 !!
 !
+! template: bgpd/bgpd.main.conf.j2
 !
 ! bgp multiple-instance
 !
-route-map FROM_BGP_SPEAKER_V4 permit 10
+! BGP configuration
 !
-route-map TO_BGP_SPEAKER_V4 deny 10
+! TSA configuration
 !
 ip prefix-list PL_LoopbackV4 permit 10.1.0.32/32
-ipv6 prefix-list PL_LoopbackV6 permit fc00:1::/64
-!
-!
-route-map TO_BGP_PEER_V4 permit 100
-!
-route-map TO_BGP_PEER_V6 permit 100
-!
-route-map FROM_BGPMON deny 10
-!
-route-map TO_BGPMON permit 10
 !
+ipv6 prefix-list PL_LoopbackV6 permit fc00:1::/64
 !
-route-map ISOLATE permit 10
-  set as-path prepend 65100
 !
-route-map set-next-hop-global-v6 permit 10
-  set ipv6 next-hop prefer-global
 !
 router bgp 65100
+!
   bgp log-neighbor-changes
-  bgp bestpath as-path multipath-relax
   no bgp default ipv4-unicast
+!
+  bgp bestpath as-path multipath-relax
+!
   bgp graceful-restart restart-time 240
   bgp graceful-restart
   bgp graceful-restart preserve-fw-state
+!
   bgp router-id 10.1.0.32
+!
   network 10.1.0.32/32
+!
   address-family ipv6
     network fc00:1::32/64
   exit-address-family
+!
   network 192.168.0.1/27
+!
+!
+!
   address-family ipv4
     maximum-paths 64
   exit-address-family
   address-family ipv6
     maximum-paths 64
   exit-address-family
-  neighbor PEER_V4 peer-group
```

Comment:
this part of configuration is not validated now. will you add tests to cover this portion?

Reply:
Yes. I'm going to add tests for this